### ISSUE TITLE
Uncaught TypeError: Cannot read property 'minimap-autohide' of undefined

### ISSUE THREAD

**Issue by libfm** [iyonaga/minimap-lens#35]
# Error when hovering the minimap
### Steps to reproduce:

1. open Atom with minimap installed and activated
2. hover the minimap
3. Everytime you hover the minimap, this error appears:

**Atom**: 1.52.0 x64
**Electron**: 6.1.12
**OS**: Debian GNU/Linux
**Thrown From**: [minimap-lens](https://github.com/iyonaga/minimap-lens) package 0.8.1

### Stack Trace

Uncaught TypeError: Cannot read property 'minimap-autohide' of undefined

```
At /home/florentin/.atom/packages/minimap-lens/lib/minimap-lens.js:294

TypeError: Cannot read property 'minimap-autohide' of undefined
    at MinimapLens.calcEditorDims (/packages/minimap-lens/lib/minimap-lens.js:294:28)
    at MinimapLens.showLens (/packages/minimap-lens/lib/minimap-lens.js:217:46)
    at /packages/minimap-lens/lib/minimap-lens.js:135:14
```

### Commands

```
     -1:37 intentions:highlight (input.hidden-input)
```

### Non-Core Packages

```
arduino-upload 1.8.0 
atom-beautify 0.33.4 
atom-material-syntax 1.0.8 
atom-material-ui 2.1.3 
atom-runner 2.7.1 
autocomplete-python 1.16.0 
busy-signal 2.0.1 
clock 1.9.1 
color-picker 2.3.0 
emmet 2.4.3 
expose 0.15.0 
file-icons 2.1.46 
git-plus 8.7.1 
git-time-machine 2.1.0 
highlight-selected 0.17.0 
intentions 1.1.5 
kite 0.197.0 
language-arduino 0.4.3 
less-than-slash 0.19.0 
linter 3.1.1 
linter-csslint 2.0.6 
linter-jsonlint 1.4.0 
linter-pylint 2.1.1 
linter-spell 0.15.0 
linter-spell-html 0.7.0 
linter-ui-default 2.3.5 
minimap 4.36.1 
minimap-lens 0.8.1 
nord-atom-syntax 0.10.0 
pigments 0.40.6 
project-viewer 1.4.0 
python-indent 1.2.6 
python-tools 0.6.9 
teletype 0.13.4 
terminal-plus 0.14.5 
```

**Comment by leommiranda** [comment]
I got the same error with minimap version 4.35.8 or higher. So I downgraded it to 4.35.7.
`apm install minimap@4.35.7`

**Comment by iyonaga** [comment]
@libfm @leommiranda 
Sorry for the late response. 
I fixed this issue in version 0.8.2.
